src/main/java/org/simulator/math/odes/RosenbrockSolver.java

```diff
@@ -296,15 +307,6 @@ public double step(DESystem DES) throws DerivativeException {
         JAC[q][j] = (-3 * g0[q] + 4 * g1[q] - g2[q]) / (2 * h);
       }
     }
-    for (int i = 0; i < numEqn; i++) {
-      for (int j = 0; j < numEqn; j++) {
-        if (i == j) {
-          I[i][j] = 1;
-        } else {
-          I[i][j] = 0;
-        }
-      }
-    }
```

Comment:
Moved to `init()`, no need to initialise the identity matrix anew in every `step()`.